src/filters.c

```diff
@@ -399,6 +399,32 @@ const struct pl_filter_function pl_filter_function_gaussian = {
     .tunable   = {true},
 };
 
+static double power_of_blackman(const struct pl_filter_ctx *f, double x)
+{
+    return pow(blackman(f, x), f->params[1]);
+}
+
+const struct pl_filter_function pl_filter_function_power_of_blackman = {
```

Comment:
Why introduce a new filter function, instead of just adding another parameter to `blackman`?

IMO that would be a much cleaner approach

Reply:
If I understand correctly you would rather to warp existing blackman() code into pow() and just leave name? If so, that's fine by me, but I would like to indicate that's not blackman rather power of blackman. At least leave comment in the code.